app/styles/main.scss

```diff
@@ -0,0 +1,207 @@
+@import "colors-fonts";
+
+:root {
+  --primary: var(--AppsilonBlue);
+  --primary-dark: var(--AppsilonNavi-strong-3);
+  --white: var(--AppsilonSnow);
+  --disabled: var(--AppsilonGray-strong-1);
+  --muted-dark: var(--AppsilonLightGray-weak-1);
+  --muted: var(--AppsilonGray-strong-1);
+}
+
+html,
+body {
+  height: 100vh;
+  color: var(--primary-dark);
+}
+
+.logos-wrapper {
+  display: flex;
+  align-items: end;
```

Comment:
The theme toggle icon is not placed in the middle.
I know adding the `align-items: center;` makes the logo not positioned at the center. Also, we need to set equal margin-top and margin-bottom for <h1> in the title to make it aligned with the RConsortium logo

Reply:
I'm glad you found this, it was bothering me too and now I was forced to correct this

Done in 1c27f28

![image](https://user-images.githubusercontent.com/211358/235189710-1f712285-00b3-4389-aab8-e36dbe0b316f.png)

Reply:
They seemed aligned to me, or do you mean something else @vedhav ?

![image](https://user-images.githubusercontent.com/211358/235189936-8fa7795d-01c1-4cf1-812e-9cf74a3c1f7c.png)